### ISSUE TITLE
Giving credit: introductions?

### ISSUE THREAD

**Issue by karenword** [carpentries/instructor-training#731]
Helpers & others involved in making workshops happen should be introduced by name & thanked for their role. This is a quick add and could go in with introductions? (But also important for wrap-up -- could go in feedback, too.)

**Comment by arokem** [comment]
Good idea! +1 for adding this to introductions. The introduction is also a good time to  point out what helpers are there for (to help!), and ask each one of them to say something (brief!) about themselves.
